pkg/apis/provisioning/v1alpha1/provisioner.go

```diff
@@ -41,6 +41,9 @@ type ProvisionerSpec struct {
 	// launched by the Provisioner. If unspecified, supports all types. Cannot
 	// be specified if label "node.kubernetes.io/instance-type" is specified.
 	InstanceTypes []string `json:"instanceTypes,omitempty"`
+	// TTLSeconds specifies the TTL for underutilized nodes.
```

Comment:
nit: let's use this field generally. I'd say "TTLSeconds determines how long to wait before attempting to terminate a node"